doc/source/conf.py

```diff
@@ -176,7 +202,6 @@ def _stop_fluent_container(gallery_conf, fname):
     },
     "navbar_end": ["version-switcher", "theme-switcher", "navbar-icon-links"],
     "navigation_depth": -1,
-    "collapse_navigation": True,
```

Comment:
@hpohekar I think this one was removed by some merges. This option was recently added to have some speedup (#1331) in the doc build.

Reply:
We have added it now. Thank you.